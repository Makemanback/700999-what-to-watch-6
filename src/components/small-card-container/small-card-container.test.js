import React from 'react';
import {Router} from 'react-router-dom';
import {render} from '@testing-library/react';
import {createMemoryHistory} from 'history';

import SmallCardContainer from './small-card-container';

const id = 1;
const image = ``;
const title = ``;
const video = ``;

describe(`Test SmallCardContainer`, () => {
  it(`SmallCardContainer should be render correctly`, () => {
    const history = createMemoryHistory();

    const {container} = render(
        <Router history={history}>
          <SmallCardContainer id={id} image={image} title={title} video={video} />
        </Router>
    );

    expect(container).toMatchSnapshot();
  });

});
