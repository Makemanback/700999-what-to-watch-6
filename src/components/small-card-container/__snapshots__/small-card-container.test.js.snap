// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test SmallCardContainer SmallCardContainer should be render correctly 1`] = `
<div>
  <article
    class="small-movie-card catalog__movies-card"
  >
    <div
      class="small-movie-card__image"
    >
      <img
        alt=""
        height="175"
        src=""
        width="280"
      />
    </div>
    <h3
      class="small-movie-card__title"
    >
      <a
        class="small-movie-card__link"
        href="/films/1"
      />
    </h3>
  </article>
</div>
`;
